### PR TITLE
chore: install wkhtmltox correctly

### PR DESCRIPTION
**Pull request by MorezMartin**
This way wkhtmltox is working with patched qt, last version avaible for debian stretch, the 0.12.5-1 doesn't seem to be supported yet

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ services:
     volumes:
       - ./conf/mariadb-conf.d:/etc/mysql/conf.d
       - /var/lib/mysql
+        #      - ./db/var/lib/mysql for data persistence
```
Comment (chabad360) [inline docker-compose.yml]: That line isn't needed, as it's already covered by 

```yaml
    - /var/lib/MySQL
```

Comment (MorezMartin) [reply docker-compose.yml]: Better this way ?

Comment (chabad360) [reply docker-compose.yml]: The less lines the better, and the less confusing

**Comment by chabad360** [comment]
Perfect! Thank you.

I will have you know that you might be waiting a while till this gets merged. The maintainer seems to be on vacation...

**Comment by MorezMartin** [comment]
Hello, no problem it is one of my first commits and i am happy to help, i
am not hurry, it is already working for me :)
Have a nice day,

**Comment by MorezMartin** [comment]
Maybe it could be cool to add some classic fonts, i have been a little bit annoyed when i wanted to get cantarell working, i may update my Dockerfile
There is a repo there https://github.com/frappe/fonts
And for me apt-install fonts-cantarell but i have had problems
Maybe i will try to make a stack.yml file too
Have a nice day

**Comment by chabad360** [comment]
Idk if you need them, it's a web app, so the site - upon loading - should automatically pull them

**Comment by MorezMartin** [comment]
Yes, for previewing in browser but not for printing PDF

**Comment by chabad360** [comment]
Oh right... Which fonts do you need?

**Comment by Thunderbottom** [comment]
0.12.5 works fine now so could you please update this PR so we can merge it in?

**Comment by MorezMartin** [comment]
Done

**Pull request merged**